CA_DataUploaderLib/BaseSensorBox.cs

```diff
@@ -93,10 +95,37 @@ static void RegisterCustomBoardCommand(IOconfMap map, CommandHandler cmd, Dictio
                 var channelWriter = channel.Writer;
                 boardCustomCommandsReceived.Add(map.McuBoard, channel.Reader);
                 cmd.AddMultinodeCommand(
-                    "custom", 
-                    a => a.Count >= 3 && a[1] == map.BoxName, 
+                    "custom",
+                    a => a.Count >= 3 && a[1] == map.BoxName,
                     a => channelWriter.TryWrite(string.Join(' ', a.Skip(2))));
             }
+            static void RegisterReconnectBoardCommand(IOconfMap map, CommandHandler cmd, Dictionary<string, TaskCompletionSource> reconnectTasks)
+            {
+                if (!map.IsLocalBoard)
+                {//if the board is not local we register the command validation with an empty action
+                    cmd.AddMultinodeCommand("reconnect", a => a.Count == 2 && a[1] == map.BoxName, _ => { });
+                    return;
+                }
+
+                if (map.McuBoard == null)
+                {
+                    cmd.AddMultinodeCommand(
+                        "reconnect", a => a.Count == 2 && a[1] == map.BoxName,
+                        _ => { CALog.LogData(LogID.A, $"reconnect failed due to missing board: {map.BoxName}"); });

```

Comment:
when we share the new feature, we need to explicitely state that this is only to reconnect boards that were detected/connected when the system was first started